openconcept/analysis/openaerostruct/aerostructural.py

```diff
@@ -1,17 +1,16 @@
 """
 This work was the basis of the following paper.
 Please cite it if you use this for your own publication!
-
-@InProceedings{Adler2022a,
-    author      = {Eytan J. Adler and Joaquim R. R. A. Martins},
-    title       = {Aerostructural wing design optimization considering full mission analysis},
-    booktitle   = {AIAA SciTech Forum},
-    doi         = {10.2514/6.2022-0382},
-    month       = {January},
-    year        = {2022}
-}
-
-Eytan Adler (Jan 2022)
+::
```

Comment:
I personally don't like having these inside docstrings unless it has some specific purpose (such as what OM does). Otherwise this should probably just go into the docs

Reply:
What does OpenMDAO do?

Reply:
They have a CLI tool that figures out which components you used, and then parses those module docstrings to provide the citation. The citation for pyOptSparse for example is stored in the pyOpt driver module docstrings. Unless you're doing something like this, it's generally better to put citation information in a doc page, since it's very rare for people who are simply using your package to be reading the source code to find out how to cite it.

Reply:
I see their citation finder now. I didn't even know about that, pretty cool. I'll add them as self.cite attributes.

Reply:
I guess what I'm saying is, if you can somehow interface it so that their cite tool also works for this, then great. Otherwise, these aren't serving any purpose in docstrings.

Reply:
Sorry, when I wrote these two your responses hadn't loaded in between. I'm gonna shoot for your final suggestion